tests/sequential/prometheus_metrics/prometheus_metrics_test.go

```diff
@@ -21,7 +21,6 @@ import (
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/kedacore/keda/v2/pkg/metricscollector"
-	. "github.com/kedacore/keda/v2/tests/helper"
```

Comment:
Don't remove this line 😄

Reply:
Damn, it was automatically removed by GoLand 😄